content/en/docs/demo/features.md

```diff
@@ -4,25 +4,24 @@ linkTitle: Features
 aliases: [/docs/demo/demo_features]
 ---
 
-- **[Kubernetes](https://kubernetes.io)**: the app is designed to run on
+- **[Kubernetes](https://kubernetes.io/)**: the app is designed to run on
   Kubernetes (both locally, as well as on the cloud) using a Helm chart.
 - **[Docker](https://docs.docker.com)**: this forked sample can also be executed
   only with Docker.
-- **[gRPC](https://grpc.io)**: microservices use a high volume of gRPC calls to
+- **[gRPC](https://grpc.io/)**: microservices use a high volume of gRPC calls to
   communicate to each other.
 - **[HTTP](https://www.rfc-editor.org/rfc/rfc9110.html)**: microservices use
   HTTP where gRPC is unavailable or not well supported.
-- **[OpenTelemetry Traces](https://opentelemetry.io)**: all services are
+- **[OpenTelemetry Traces](/docs/concepts/signals/traces/)**: all services are
```

Comment:
Note the change in URL from the site homepage to the Traces page under Concepts.